src/lib.rs

```diff
@@ -536,6 +536,10 @@ impl ToTokens for Member {
     }
 }
 
+impl ToTokens for Order {
+    fn to_tokens(&self, _tokens: &mut TokenStream) {}
```

Comment:
Why did you implement ToTokens on Order? This will never be converted to tokens...

Reply:
Removed.